### PR TITLE
🌐  Add Korean translation for docs/ko/docs/tutorial/query-params-str-validations.md

### PR DESCRIPTION
**Pull request by eeHoon**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4a8e45c257f0b0bd5e80e327c3a84c1657054b3b at: https://4002e286.fastapitiangolo.pages.dev

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 1]
```diff
@@ -0,0 +1,967 @@
...
+    {!> ../../../docs_src/query_params_str_validations/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+"
+
+    ```Python hl_lines="9"
+    {!> ../../../docs_src/query_params_str_validations/tutorial001.py!}
+    ```
+
+쿼리 매개변수 `q`는 `Union[str, None]` 유형 (Python 3.10에서는 `str | None`)입니다.
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:19]:
```suggestion
쿼리 매개변수 `q`는 `Union[str, None]` 타입 (Python 3.10에서는 `str | None`)입니다.
```

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 2]
```diff
@@ -0,0 +1,967 @@
...
+이것은 `str` 유형이지만 `None`이 될 수 있다는 것을 의미하며, 실제로 기본값은 `None`이므로
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:20]:
```suggestion
이것은 `str` 타입이지만 `None`이 될 수 있다는 것을 의미하며, 실제로 기본값은 `None`이므로
```

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 3]
```diff
@@ -0,0 +1,967 @@
...
+    ```
+
+!!! info
+    FastAPI 버전 0.95.0에서 `Annotated`를 지원에 추가하고 권장하기 시작했습니다.
+
+    이전 버전을 사용하면 `Annotated`을 사용하려고 할 때 오류가 발생합니다.
+
+    `Annotated`를 사용하기 전에 [FastAPI 버전을 업그레이드](../deployment/versions.md#upgrading-the-fastapi-versions){.internal-link target=_blank}하여 최소한 0.95.1로 업그레이드하세요.
+
+## `q` 매개변수의 유형에 `Annotated` 사용하기
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:64]:
```suggestion
## `q` 매개변수의 타입에 `Annotated` 사용하기
```

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 4]
```diff
@@ -0,0 +1,967 @@
...
+
+    !!! tip
+        가능한 경우 'Annotated' 버전을 사용하십시오.
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/query_params_str_validations/tutorial005.py!}
+    ```
+
+!!! note
+    어떤 유형의 기본값이든 (None 포함) 매개변수를 선택사항(optional)으로 만듭니다.
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:389]:
```suggestion
    어떤 타입의 기본값이든 (None 포함), 기본값을 가진다면, 해당 매개변수는 선택사항(optional)이 됩니다.
```

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 5]
```diff
@@ -0,0 +1,967 @@
...
+    Pydantic 과 FastAPI 에서 값을 명시적으로 필수로 선언하기 위해 사용됩니다.
+
+이렇게 하면 **FastAPI** 는 이 매개변수가 필수임을 알게 됩니다.
+
+## `None`을 쓰면서 필수선언하기
+
+매개변수가 `None`을 허용할 수 있지만 여전히 필수로 선언할 수 있습니다.
+이렇게 하면 클라이언트가 값이 `None`이더라도 값을 보내도록 강제합니다.
+
+이를 위해 `None`이 유효한 유형임을 선언하고 기본값으로 `...`을 사용할 수 있습니다.
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:487]:
```suggestion
이를 위해 `None`이 유효한 타입임을 선언하고 기본값으로 `...`을 사용할 수 있습니다.
```

File: docs/ko/docs/tutorial/query-params-str-validations.md [hunk 6]
```diff
@@ -0,0 +1,967 @@
...
+
+문자열에 대한 검증:
+
+* `min_length`
+* `max_length`
+* `pattern`
+
+이 예에서는 `str` 값에 대한 검증을 선언하는 방법을 보았습니다.
+
+숫자와 같은 다른 유형에 대한 검증을 선언하는 방법을 보려면 다음 장을 참조하십시오.
```
Comment (junah201) [inline docs/ko/docs/tutorial/query-params-str-validations.md:967]:
```suggestion
숫자와 같은 다른 타입에 대한 검증을 선언하는 방법을 보려면 다음 장을 참조하십시오.
```

**Review by junah201 — COMMENTED** [review]
Type is already being translated into `타입` in more than 18 documents, it it better to use `타입` instead of `유형` 

이미 18개 이상의 문서에서 type을 타입으로 변역하고 있으므로, 유형 대신에 type을 사용하는 것이 더 좋을 것 같습니다.

**Comment by tiangolo** [comment]
It seems this was covered in another PR, so I'll pass on this one. But thanks for your effort! :coffee:

**Pull request closed without merging**